src/Build/BackEnd/Node/OutOfProcNode.cs

```diff
@@ -583,10 +583,8 @@ private void SendPacket(INodePacket packet)
         {
             if (_nodeEndpoint.LinkStatus == LinkStatus.Active)
             {
-#if RUNTIME_TYPE_NETCORE
```

Comment:
Wait I'm confused by this part. If it was only firing on core before, is it right to do it now? Or had we completely removed even the opt-in part on core, so this is now just unifying that behavior?

Reply:
For some reason the logic was bit convoluted - on sender side there was an error logged instead of transmitting the event - but just for core. On the receiving side there was a warning logged if the event was received. The warning was opt-out - it was logged by default.

So this was opt-out - now the opt-out mechanism is removed.
The severity of the diagnostic for the NetFx will effectively rise from warning to error - which might break scenarios where ppl actively avoided addressing the issue - e.g.: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1936439 - which might be a good forcing function, but possibly unappreciated breakage.


How about we keep this a warning for NETFx?

Reply:
As discussed offline, we will keep warning for .NETFx an .NET version will produce error.